File: Bridge/System/Threading/Tasks/UnobservedTaskExceptionEventArgs.cs
using Bridge;

namespace System.Threading.Tasks
{
    public class UnobservedTaskExceptionEventArgs : EventArgs
    {
        public UnobservedTaskExceptionEventArgs(AggregateException exception)
        {
        }

        public AggregateException Exception
        {
            get
            {
                throw new NotImplementedException();
            }
        }

        public bool Observed
        {
            get
            {
                throw new NotImplementedException();
            }
        }

        public void SetObserved()
        {
            throw new NotImplementedException();
        }
    }
}

File: Bridge/Resources/.generated/system/threading/tasks/unobservedTaskExceptionEventArgs.js
    Bridge.define("System.Threading.Tasks.UnobservedTaskExceptionEventArgs", {
        props: {
            Exception: {
                get: function () {
                    throw new System.NotImplementedException();
                }
            },
            Observed: {
                get: function () {
                    throw new System.NotImplementedException();
                }
            }
        },
        ctors: {
            ctor: function (exception) {
                this.$initialize();
            }
        },
        methods: {
            setObserved: function () {
                throw new System.NotImplementedException();
            }
        }
    });
